canonical-web-and-design/ubuntu.com
# Update location of brandmark zip files

## Done

- Update links to brandmarks on the assets server

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/blog/press-centre
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the logo packs download

## Issue / Card

Fixes #5405

## Screenshots

![image](https://user-images.githubusercontent.com/441217/60418648-5f44d380-9bdb-11e9-95b4-5e338404677d.png)
